guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,21 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Don't use string interpolation
+
+By using string interpolation it is impossible for logging frameworks that support structure logging to distinguish values within the string. We must use the normal argument based overload so logging frameworks like Serilog can capture structured data.
```

Comment:
@SimonCropp correct, that is why the concatenation will not work and only arguments. In your sample you even name your argument which is a serilog only feature that doesn't work with other loggers. However, Serilog works with regular argument index identifiers too although then the arguments do not have a actual name.